ansible/roles/swap/tasks/main.yml

```diff
@@ -48,7 +48,7 @@
 - name: Set swappiness
   sysctl:
     name: vm.swappiness
-    value: 0
+    value: 1
```

Comment:
@emord Yes. I wanted to check that it was working as expected on softlayer, which I can now confirm.